back-end/src/sciler/handler/handler.go

```diff
@@ -313,6 +341,20 @@ func (handler *Handler) onInstructionMsg(raw Message) {
 					for _, timer := range handler.Config.Timers {
 						handler.SendStatus(timer.ID)
 					}
+					handler.SendEventStatus()
+				}
+			case "name":
```

Comment:
This instruction should also be added to the message manual.
Maybe rename this instruction to something more informative such as `get name` or `send name`.